neon_mq_connector/connector.py

```diff
@@ -214,3 +222,53 @@ def stop_consumers(self, names: tuple = ()):
                     self.consumers[name].join()
             except Exception as e:
                 raise ChildProcessError(e)
+
+    def sync(self, vhost: str = None, exchange: str = None, queue: str = None, request_data: dict = None):
+        """
+            Periodical notification message to be sent into MQ
```

Comment:
Is there a default listener for this message? I think the docstring here should describe more what this is for.

i.e.
```
Periodically sends an MQ message to notify other services that this service is active.
```